src/styles/custom.scss

```diff
@@ -153,15 +153,22 @@ iframe {
   box-shadow: 10px 10px 20px rgba(0, 0, 0, 0.1);
 }
 
-.git-link svg {
-  height: 1em;
-  width: 1em;
-  vertical-align: -0.1em;
-}
-.git-link path {
-  fill: currentColor;
+.git-link {
+  &::before {
+    content: '';
+    height: 1em;
+    width: 1em;
+    display: inline-block;
+    background-color: currentColor;
+    mask-repeat: no-repeat;
+    mask-position: center;
+    mask-size: 100% 100%;
+    mask-image: url(../../static/icons/code-branch.svg);
+    vertical-align: -0.1em;
```

Comment:
The alignment is slightly off when compared with `main`:

| main | branch |
| - | - |
| ![Screenshot 2024-01-09 at 9 31 15 AM](https://github.com/ionic-team/ionic-docs/assets/2721089/a1f80f47-fd51-40bb-b724-d6a6c5f5cf0e) | ![Screenshot 2024-01-09 at 9 31 12 AM](https://github.com/ionic-team/ionic-docs/assets/2721089/33d09b98-df8a-4be8-8390-fbca799fda28) |

Reply:
Fixed: https://github.com/ionic-team/ionic-docs/pull/3322/commits/b5ea741137d36061e20b4045fbda68e22337e3dd